File: pgm/ABC_R/ABC_216_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
dot <- "\\."
minus <- "-"
plus <- "+"
two <- 2
six <- 6
X <- strsplit(arr[1], split = dot)[[1]][1] |>
    as.integer()
Y <- strsplit(arr[1], split = dot)[[1]][2] |>
    as.integer()
if (Y <= two) {
    paste0(X, minus) |>
        cat()
} else if (Y <= six) {
    X |>
        cat()
} else {
    paste0(X, plus) |>
        cat()
}
cat("\n")
q("no")

File: pgm/ABC_Dart/ABC_216_A.dart
// import 'dart:collection';
// import 'dart:convert';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  const String minus = '-';
  const String dot = '.';
  const String plus = '+';
  const String blank = '';
  const int two = 2;
  const int six = 6;
  final [int X, int Y] = stdin
      .readLineSync()!
      .split(dot)
      .map(int.parse)
      .toList();
  final String sign = switch (Y) {
    <= two => minus,
    <= six => blank,
    _ => plus,
  };
  print("$X$sign");
  exit(0);
}
